apps/eth-wrapper/src/utils/Erc20Constants.ts

```diff
@@ -0,0 +1,11 @@
+export const Erc20 = [
+    "function approve(address _spender, uint256 _value) public returns (bool success)",
+    "function allowance(address _owner, address _spender) public view returns (uint256 remaining)",
+    "function balanceOf(address _owner) public view returns (uint256 balance)",
+    "event Approval(address indexed _owner, address indexed _spender, uint256 _value)"
+];
+
+export function getWethAddress(network: string): string {
+    return network === "mainnet" ? "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2" :
```

Comment:
I would use a switch case or mapping and make it so that it fails if it is an unknown network. Currently on all "non-mainnet" networks we would use the rinkeby address ... which would cause loss of funds on networks like xdai.